library/account/src/main/kotlin/org/cru/godtools/account/AccountModule.kt

```diff
@@ -27,5 +33,14 @@ abstract class AccountModule {
             override fun userId() = accountManager.userId
             override suspend fun authenticate() = accountManager.authenticateWithMobileContentApi().getOrNull()
         }
+
+        const val IS_CONNECTED_STATE_FLOW_ACCOUNTS = "STATE_FLOW_IS_CONNECTED_ACCOUNTS"
+
+        @Provides
+        @Reusable
+        @Named(IS_CONNECTED_STATE_FLOW_ACCOUNTS)
+        fun isConnectedStateFlow(@ApplicationContext context: Context, scope: CoroutineScope) =
+            context.isConnectedFlow()
+                .stateIn(scope, WhileSubscribed(5_000), false)
```

Comment:
why did you duplicate this here instead of just use the `IS_CONNECTED_STATE_FLOW` that is already on the dagger graph?

Reply:
@frett Based on my understanding, I would have had to implement the base module into the account module, and I didn't think you would have wanted that.

Reply:
the account module is already dependent on `:library:base` via the path: `account -> api -> model -> base`.

The symbols are just not present because `model` defines an `implementation` dependency and not an `api` dependency, which is correct because model doesn't need to expose the base dependency because nothing in base is part of the model public API.

Adding an `implementation` dependency from account to `base` would be the right thing here since the `IS_CONNECTED_STATE_FLOW` is part of the public API of the base module, and you want to use it.

The reason this wasn't the case with the `FilterMenuItem` from a previous PR is because that was not meant to be part of the public API that other modules/features can use

Reply:
the difference between an `implementation` and an `api` dependency is whether the transitive dependency is exposed as part of the public API of the module you are adding the dependency to.